app/angular/AngularView/src/app/booking-details/booking-details.component.ts

```diff
@@ -183,7 +183,7 @@ export class BookingDetailsComponent implements OnInit {
       this.bookingsService.createBooking(finalData).subscribe(
         (data: any) => {
           this.inputGroup2.items.forEach(element => {
-            const finalItemData = { booking_source: data.id, item: element.item, quantity: element.quantity, status: 'PEN' };
+            const finalItemData = { booking_source: data.id, item: element.item, quantity: element.quantity, status: 'UNC' };
```

Comment:
sorry for inconvenience, this original line is correct: 'UNC' should be replaced with 'PEN'

Reply:
but for testing purposes u may change line 176 to 'PEN' since like i've said u have no access to the confirmation emails